1.data-exploration/scripts/nbconverted/1.model_data_visualization.py

```diff
@@ -189,9 +173,9 @@
 age_distribution_plot = (
     gg.ggplot(model_df, gg.aes(x="age_distribution"))
     + gg.geom_density()
-    + gg.geom_vline(xintercept=adult_threshold, linetype="dashed", color="red")
+    + gg.geom_vline(xintercept=18, linetype="dashed", color="red")
```

Comment:
I'm wondering why you removed this variable? Why hardcode 18 instead of keeping `adult_threshold`?

Reply:
Also, does the dotted line mean anything anymore? (I remember you saying the category is now defined)